templates/climo.bash

```diff
@@ -120,12 +120,23 @@ if [ $? != 0 ]; then
   exit 2
 fi
 
-# Move regridded climo files to final destination
+# Move regridded climo files to final destination 
+{% if frequency == 'monthly' %}
 {
   dest={{ output }}/post/{{ component }}/{{ grid }}/clim/{{ '%dyr' % (yr_end-yr_start+1) }}
   mkdir -p ${dest}
   mv output/*.nc ${dest}
 }
+
+{% elif frequency.startswith('monthly_diurnal') %}
+{
+  dest={{ output }}/post/{{ component }}/{{ grid }}/clim_{{ subsection }}/{{ '%dyr' % (yr_end-yr_start+1) }}
```

Comment:
`subsection` also includes `grid`. I could try to remove the `grid` part in `climo.py` but it might be useful to just have the directory name match `subsection`, as this code does.